presto-main/etc/config.properties

```diff
@@ -51,6 +51,7 @@ plugin.bundles=\
   ../presto-node-ttl-fetchers/pom.xml,\
   ../presto-hive-function-namespace/pom.xml,\
   ../presto-delta/pom.xml,\
+  ../presto-mongodb/pom.xml,\
```

Comment:
This and the mongodb.properties file is not needed

Reply:
@jaystarshot reverted the files